keycloak/keycloak-ui
# Fixed SAML details header and listing providers by name - #1942

## Motivation
Closes #1942 & the issue with listing identity providers by displayName instead of the alias. This is now working as per old admin UI.

## Verification Steps
1. Go to `Identity providers >> Add SAML provider`.
2. Enter `new SAML provider` as the display name and click on the `Save` button.
3. Navigate to all identity providers.
4. Verify that the newly create provider is listed by name and not an alias. 
5. Click on the provider and verify that the display name is showing in the header instead of the alias.

## Checklist:
